package exception;

public class DemoException2 
{
   public static void main(String args[])
		{
			int abc[]= {10,20,30,40,50};
			
		
		
	try
	{
		
			

	}
	catch(Array e)

	{
		System.out.println("Exception Occur");
	}

	}
	}

}
